<launch>
  <!-- start the controllers -->
  <include file="$(find tams_pr2_mannequin_mode)/launch/controllers.launch"/>

  <!-- Start the scripts to actually lock the arms and head in place -->
  <include file="$(find tams_pr2_mannequin_mode)/launch/trajectory_lock.launch"/>

  <node pkg="tams_pr2_mannequin_mode" name="mannequin_mode_server" type="mannequin_mode_server.py" output="screen">
    <rosparam param="pr2_controllers">["head_traj_controller", "l_arm_controller", "r_arm_controller"]</rosparam>
    <rosparam param="loose_controllers">["head_traj_controller_loose", "l_arm_controller_loose", "r_arm_controller_loose"]</rosparam>
  </node>

</launch>
